app/config/permissions.js

```diff
@@ -287,6 +287,10 @@ const groups = [
       ],
       full: [
         'manage-signatures',
+        'manage-only-specific-signatures',
```

Comment:
do these 2 play well together? the "manage-signatures"  > you can manage all of them.
"manage-only-specific-signatures" > you can only manage yours